blue_bringup/launch/include/single_arm_control.launch.xml

```diff
@@ -35,6 +35,8 @@
       output="screen">
       <rosparam param="start_controllers">
         - blue_controllers/joint_state_controller
+        - blue_controllers/joint_trajectory_controller
```

Comment:
I don't think it makes sense to start these controllers by default; the launch files in blue_bringup are used by a bunch of different applications which don't necessarily use these controllers.

Perhaps you could make a separate launch file that (a) includes the core bringup launchfile and (b) creates a controller spawner for the controllers you need?

Reply:
How about I create a separate launch.xml file and reference that in a modified bringup launchfile?

Reply:
Anyway, I've reverted that commit.